### ISSUE TITLE
:vertical Man doesn't split vertically

### ISSUE THREAD

**Issue by joshtriplett** [vim/vim#3249]
`:vertical Man somemanpage` still does a horizontal split. It uses the `new` command internally, but `:vertical` doesn't seem to affect it.

```
VIM - Vi IMproved 8.1 (2018 May 17, compiled Jun 20 2018 02:44:05)
Included patches: 1-89
```

**Comment by brammool** [comment]
> `:vertical Man somemanpage` still does a horizontal split. It uses the `new` command internally, but `:vertical` doesn't seem to affect it.

That plugin is maintained by SungHyun Nam, I'll cc him.

-- 
I'd like to meet the man who invented sex and see what he's working on now.

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///

**Comment by vim-ml** [comment]
Hi,

On Tue, Jul 24, 2018 at 11:59 AM, Josh Triplett
<vim-dev-github@256bit.org> wrote:
> :vertical Man somemanpage still does a horizontal split. It uses the new
> command internally, but :vertical doesn't seem to affect it.
>
> VIM - Vi IMproved 8.1 (2018 May 17, compiled Jun 20 2018 02:44:05)
> Included patches: 1-89
>

You can try applying the attached patch to the man.vim plugin.

- Yegappan

**Comment by joshtriplett** [comment]
The patch doesn't seem to have made it through to Github. I dug the following out of the mailing list archive:

[man.diff.txt](https://github.com/vim/vim/files/2226671/man.diff.txt)

That patch does seem to work.

**Comment by vim-ml** [comment]
2018-07-25 오후 2:32에 Yegappan Lakshmanan 이(가) 쓴 글:
> Hi,
> 
> On Tue, Jul 24, 2018 at 11:59 AM, Josh Triplett
> <vim-dev-github@256bit.org> wrote:
>> :vertical Man somemanpage still does a horizontal split. It uses the new
>> command internally, but :vertical doesn't seem to affect it.
>>
>> VIM - Vi IMproved 8.1 (2018 May 17, compiled Jun 20 2018 02:44:05)
>> Included patches: 1-89
>>
> 
> You can try applying the attached patch to the man.vim plugin.

Thanks!  I'll send updated man.vim plugin to Bram.

- namsh

**Comment by vim-ml** [comment]
Hi,

On Tue, Jul 24, 2018 at 11:06 PM, SungHyun Nam <goweol@gmail.com> wrote:
> 2018-07-25 오후 2:32에 Yegappan Lakshmanan 이(가) 쓴 글:
>>
>> Hi,
>>
>> On Tue, Jul 24, 2018 at 11:59 AM, Josh Triplett
>> <vim-dev-github@256bit.org> wrote:
>>>
>>> :vertical Man somemanpage still does a horizontal split. It uses the new
>>> command internally, but :vertical doesn't seem to affect it.
>>>
>>> VIM - Vi IMproved 8.1 (2018 May 17, compiled Jun 20 2018 02:44:05)
>>> Included patches: 1-89
>>>
>>
>> You can try applying the attached patch to the man.vim plugin.
>
>
> Thanks!  I'll send updated man.vim plugin to Bram.
>
> - namsh
>

I am attaching a simplified diff.

Regards,
Yegappan

**Comment by chrisbra** [comment]
fixed by 91f84f6e11cd879d43d651c0903d85bff95f0716
